projects/jsf/src/lib/form-content/section/section.component.html

```diff
@@ -20,7 +20,7 @@
        [title]="formItem.helpText"
        [src]="'assets/jsf-images/info.svg'">
 </div>
-<div class="description">{{formItem.description}}</div>
+<div [ngClass]="{ 'collapsible-sections-padding': config.enableCollapsibleSections }" class="description">{{formItem.description}}</div>
```

Comment:
In case you didn't know, you can also do this, which seems a bit simpler to me:
```
<div [class.collapsible-sections-padding]="config.enableCollapsibleSections" class="description">{{formItem.description}}</div>
```

Reply:
Done!